fence/agents/gce/fence_gce.py

```diff
@@ -5,7 +5,8 @@
 sys.path.append("@FENCEAGENTSLIBDIR@")
 
 import googleapiclient
-import oauth2client
+import googleapiclient.discovery
```

Comment:
This is already imported in line 7.

Reply:
importing the package doesn't mean it imported the module and I get an error that googleapiclient.discovery was not found. What I can do is to remove the line 7 and leave just the "import googleapiclient.discovery"